hudi-client/src/main/java/org/apache/hudi/io/HoodieCommitArchiveLog.java

```diff
@@ -290,7 +291,7 @@ private IndexedRecord convertToAvroRecord(HoodieTimeline commitTimeline, HoodieI
       case HoodieTimeline.CLEAN_ACTION: {
         if (hoodieInstant.isCompleted()) {
           archivedMetaWrapper.setHoodieCleanMetadata(CleanerUtils.getCleanerMetadata(metaClient, hoodieInstant));
-        } else {
+        } else if (HoodieInstant.State.REQUESTED != hoodieInstant.getState()) {
```

Comment:
Are you sure the Cleaner requested file is empty ? I quickly looked at https://github.com/apache/incubator-hudi/blob/master/hudi-client/src/main/java/org/apache/hudi/HoodieCleanClient.java#L124 and does not appear to be so. Can you double check ?